samples/lorawan/class_a/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_LOG=y
+CONFIG_SPI=y
+CONFIG_GPIO=y
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Why do we need newlib?

Reply:
Some functions used by the stack are not present in the minimal libc.

Reply:
@jukkar The reason is provided by @alexanderwachter